runtime/drive_handler.go

```diff
@@ -111,6 +111,9 @@ func (h *stubDriveHandler) createStubDrive(driveID, path string) error {
 			h.logger.WithError(err).Errorf("unexpected error during %v close", f.Name())
 		}
 	}()
+	if err := f.Chmod(0600); err != nil {
```

Comment:
Do we need this? The file is created with 0600 above.

Reply:
I believe that just opens the file with `0600` permissions. If I remove that line, I get a permission denied error.

Reply:
Looks like this was prior to me adding the `chownr` function, so this goes away.